tests/iface_loopback_action/test_iface_loopback_action.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+import logging
+import random
+from tests.common.reboot import reboot
+from tests.common.config_reload import config_reload
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+from tests.common.plugins.allure_wrapper import allure_step_wrapper as allure
+from .iface_loopback_action_helper import ACTION_FORWARD, ACTION_DROP, NUM_OF_TOTAL_PACKETS
+from .iface_loopback_action_helper import verify_traffic
+from .iface_loopback_action_helper import config_loopback_action
+from .iface_loopback_action_helper import clear_rif_counter
+from .iface_loopback_action_helper import verify_interface_loopback_action
+from .iface_loopback_action_helper import verify_rif_tx_err_count
+from .iface_loopback_action_helper import shutdown_rif_interfaces, startup_rif_interfaces
+from tests.common.platform.interface_utils import check_interface_status_of_up_ports
+
+
+pytestmark = [pytest.mark.topology('any')]
+
+logger = logging.getLogger(__name__)
+allure.logger = logger
+
+
+def test_loopback_action_basic(duthost, ptfadapter, ports_configuration):
+    rif_interfaces = list(ports_configuration.keys())
+    intf_count = len(rif_interfaces)
+    with allure.step("Verify the rif loopback action default action: loopback traffic will be forwarded"):
```

Comment:
Will this still work if allure is not configured?

Reply:
Yes, it works, and it is not the first time we use the allure in sonic-mgmt, such as in tests/ecmp/inner_hashing/test_inner_hashing_lag.py, we already used the allure.